packages/swapper/src/swappers/cow/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -112,10 +112,6 @@ export async function getCowSwapTradeQuote(
       },
     } = quoteResponse
 
-    const quoteSellAmountPlusFeesCryptoBaseUnit = bnOrZero(sellAmountCryptoBaseUnit).plus(
```

Comment:
was this change intentional? would be cautious of changing implementation in polygon feature addition

Reply:
Definitely not, I'm messing around with IDE's and my workflow a bit. Fixed